source/tutorials/sparql.md

```diff
@@ -9,7 +9,7 @@ language through examples but does not aim to be complete.
 If you are looking for a short introduction to SPARQL and Jena try
 [Search RDF data with SPARQL](http://www.ibm.com/developerworks/xml/library/j-sparql/).  If you are looking to execute SPARQL queries in code and already known SPARQL then you likely want to read the [ARQ Documentation][1] instead.
 
-[1]: https://jena.apache.org/documentation/query/index.html
+[1]: /documentation/query/index.html
```

Comment:
Tested this one locally, and worked fine. It was also a good opportunity to set up `hugo` on my laptop. Assuming others will work too.

Reply:
The only issue I have with Hugo ATM is that it does not add ".html" to a link like the deployed website does.